internal/testdata/test_proto/test_proto_v2.proto

```diff
@@ -60,7 +62,18 @@ message MessageV2 {
 
   optional bytes f_bytes = 19;
 
-  // NEXT: 20
+  optional google.protobuf.BoolValue f_BoolValue = 20;
+  optional google.protobuf.StringValue f_StringValue = 21;
+  optional google.protobuf.DoubleValue f_DoubleValue = 22;
+  optional google.protobuf.Int32Value f_Int32Value = 23;
+  optional google.protobuf.Int64Value f_Int64Value = 24;
+  optional google.protobuf.BytesValue f_BytesValue = 25;
+  optional google.protobuf.UInt32Value f_Uint32Value = 26;
+  optional google.protobuf.UInt64Value f_Uint64Value = 27;
+
+  repeated google.protobuf.StringValue r_StringValue = 28;
```

Comment:
Should this be `optional`?

Reply:
I'm not super familiar with proto2 but I think a field can only be one of `optional`, `required` or `repeated` https://developers.google.com/protocol-buffers/docs/proto

Reply:
Under "[Updating a Message Type](https://developers.google.com/protocol-buffers/docs/proto#updating)," it says
> For string, bytes, and message fields, optional is compatible with repeated

So I think is should be okay to make this field `optional` as well.

Reply:
nvm - I think I misunderstood the docs.